### ISSUE TITLE
Suggest best compatible image browser

### ISSUE THREAD

**Issue by rasikamehra** [lllyasviel/stable-diffusion-webui-forge#1274]
Can someone please suggest best compatible Image Browser for forge GUI? Tried a couple of them, none of them works as expected. Which image browser do you use with forge GUI?

**Comment by Dragoy** [comment]
was looking for it too, ended up using the old classic explorer.exe

**Comment by supersonic13** [comment]
This worked for me after a couple of updates from the DEV .. https://github.com/AlUlkesh/stable-diffusion-webui-images-browser

**Comment by YakuzaSuske** [comment]
Best in my opinion is Infinite image browser the UI is very extensive and you get lots of options to look through your outputs.
https://github.com/zanllp/sd-webui-infinite-image-browsing

**Comment by rasikamehra** [comment]
> This worked for me after a couple of updates from the DEV .. https://github.com/AlUlkesh/stable-diffusion-webui-images-browser

Tried this, it slows down the GUI and make it laggy noticeably. Also, with latest Forge updates, this extension does not work.

**Comment by rasikamehra** [comment]
> Best in my opinion is Infinite image browser the UI is very extensive and you get lots of options to look through your outputs. https://github.com/zanllp/sd-webui-infinite-image-browsing

Thanks for the suggestion but already tried this. It makes the GUI very laggy and does not work well with mobile view.

**Comment by supersonic13** [comment]
> > This worked for me after a couple of updates from the DEV .. https://github.com/AlUlkesh/stable-diffusion-webui-images-browser
> 
> Tried this, it slows down the GUI and make it laggy noticeably. Also, with latest Forge updates, this extension does not work.

The laggy behaviour was due to a lot of network calls - i..e, "queue/join" method that was called - something to do with the Gradio update. The DEV definitely fixed it .. so it is possibly from some other extension that you're using ?

**Comment by rasikamehra** [comment]
> > > This worked for me after a couple of updates from the DEV .. https://github.com/AlUlkesh/stable-diffusion-webui-images-browser
> > 
> > 
> > Tried this, it slows down the GUI and make it laggy noticeably. Also, with latest Forge updates, this extension does not work.
> 
> The laggy behaviour was due to a lot of network calls - i..e, "queue/join" method that was called - something to do with the Gradio update. The DEV definitely fixed it .. so it is possibly from some other extension that you're using ?

Is this image browser extension working for you? I am using the latest version of forge and this extension seems to be broken. Images are not loading and all button on this extension are disabled.

**Comment by supersonic13** [comment]
> > > > This worked for me after a couple of updates from the DEV .. https://github.com/AlUlkesh/stable-diffusion-webui-images-browser
> > > 
> > > 
> > > Tried this, it slows down the GUI and make it laggy noticeably. Also, with latest Forge updates, this extension does not work.
> > 
> > 
> > The laggy behaviour was due to a lot of network calls - i..e, "queue/join" method that was called - something to do with the Gradio update. The DEV definitely fixed it .. so it is possibly from some other extension that you're using ?
> 
> Is this image browser extension working for you? I am using the latest version of forge and this extension seems to be broken. Images are not loading and all button on this extension are disabled.

Yes. I’ve updated forge to the latest too and it works perfectly now. Maybe re-natal the extension ? Or even update it and try maybe ?
